libminifi/include/SchedulingAgent.h

```diff
@@ -28,72 +28,90 @@
 #include <algorithm>
 #include <thread>
 #include "utils/TimeUtil.h"
-#include "Logger.h"
+#include "core/logging/Logger.h"
 #include "Configure.h"
 #include "FlowFileRecord.h"
-#include "Logger.h"
-#include "Processor.h"
-#include "ProcessContext.h"
+#include "core/logging/Logger.h"
+#include "core/Processor.h"
+#include "core/ProcessContext.h"
+
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+
 
 //! SchedulingAgent Class
-class SchedulingAgent
-{
-public:
-	//! Constructor
-	/*!
-	 * Create a new processor
-	 */
-	SchedulingAgent() {
-		configure_ = Configure::getConfigure();
-		logger_ = Logger::getLogger();
-		_running = false;
-	}
-	//! Destructor
-	virtual ~SchedulingAgent()
-	{
+class SchedulingAgent {
+ public:
+  //! Constructor
+  /*!
+   * Create a new processor
+   */
+  SchedulingAgent(std::shared_ptr<provenance::ProvenanceRepository> repo) {
+    configure_ = Configure::getConfigure();
+    logger_ = logging::Logger::getLogger();
+    _running = false;
```

Comment:
Can you move "_" to end of class data member to adhere to Google C++ Style guide? Same for all variables here.